hasura/migrations/default/1666579812519_alter_table_public_organizations_add_column_sandbox/down.sql

```diff
@@ -0,0 +1,4 @@
+-- Could not auto-generate a down migration.
+-- Please write an appropriate down migration for the SQL below:
```

Comment:
two migration files

Reply:
can we delete the second migration file and take the down migration and put it here.